src/platform/apollolake/include/platform/lib/memory.h

```diff
@@ -166,7 +166,7 @@
 
 /* HP SRAM Heap */
 #define HEAP_HP_BUFFER_BASE	HP_SRAM_BASE
-#define HEAP_HP_BUFFER_SIZE	0x8000
+#define HEAP_HP_BUFFER_SIZE	0xA000
```

Comment:
If this adjustment is required by any of the preceding commits, perhaps should go before that one, not after.

Reply:
It's before, but for some reason github shows it in the wrong order.